sdk/libs-haskell/da-hs-base/src/Options/Applicative/Extended.hs

```diff
@@ -45,7 +46,13 @@ determineAutoM m = \case
 -- This maps yes to "Just true", no to "Just False" and auto to "Nothing"
 flagYesNoAuto' :: String -> String -> Mod OptionFields YesNoAuto -> Parser YesNoAuto
 flagYesNoAuto' flagName helpText mods =
-    optionOnce reader (long flagName <> value Auto <> help helpText <> completeWith ["true", "false", "yes", "no", "auto"] <> mods)
+    flagYesNoAuto'' flagName helpText (value Auto <> mods)
+
+-- | This constructs flags that can be set to yes, no, or auto, with no default
+-- This maps yes to "Just true", no to "Just False" and auto to "Nothing"
+flagYesNoAuto'' :: String -> String -> Mod OptionFields YesNoAuto -> Parser YesNoAuto
+flagYesNoAuto'' flagName helpText mods =
```

Comment:
I don't love the increasing number of ticks...
Could you at least update the comment, and perhaps consider if this can be implemented at call site?